src/array.jl

```diff
@@ -154,14 +154,23 @@ CategoricalMatrix{T}(m::Int, n::Int; ordered=false) where {T} =
 
 ## Constructors from arrays
 
-# This method is needed to ensure that a copy of the pool is always made
+# This method is needed to ensure that a copy of the refs and pool is always made
 # so that ordered!() does not affect the original array
```

Comment:
Just remove this line, now we make a copy because that's what the function is documented to do.

Reply:
fixed